core/domain/feedback_domain.py

```diff
@@ -37,16 +37,32 @@ class FeedbackThread(python_utils.OBJECT):
         summary: str. A summary of the feedback thread.
         has_suggestion: bool. Whether the feedback thread includes a
             suggestion.
+        message_count: int. The number of messages posted onto the thread.
         created_on: datetime.datetime. The time when the feedback thread was
             created.
         last_updated: datetime.datetime. The time when the feedback thread
             was last updated.
+        updated_status: str. The change in status since the last message, if
+            any.
+        last_message_id: int. The id of the last message in the thread.
+        last_message_text: str. The text of the last message in the thread.
+        last_message_author_id: str. The author id of the last message in the
```

Comment:
Will the author ID of the last message be displayed in the UI?

Reply:
Nope, I've removed it and stored the username directly.

Reply:
Hm, similar question though: will the username be displayed in the UI?

Reply:
Yes, it will be shown in the improvements card as:

> Last Message:
> message_text
> **by: message_author**

Reply:
Same thing as mentioned above -- maybe it's better to have this as a last_nonempty_message_author field.

Reply:
Hmm well this is trickier since author ids are allowed to be `None` (for anonymous responses). I think it's fine to only have last_message_author/last_message_text for now, we can figure out whether to show a status-change signal and other helpful info from just those two.

Reply:
Sorry, just to be clear -- if the last message is just a status change (with no text), what are you considering to be the "last message" in "last_message_author/last_message_text"?